README.md

```diff
@@ -80,3 +80,16 @@ python-pil
 python-reportlab
 python-cairo
 ```
+
+
+### Running unittests
+
+For developers or packagers, there are unittests. To run the unittest,
+proceed as follows:
+
+```
+pip install pytest
+pip install .
+cd unittests
+pytest
+```
```

Comment:
We haven't used pytest in UniConvertor. All the tests are unittests using built-in unittest package. The pytests are introduced in color-picker app recently because we start porting to py3 some parts of Uniconvertor,

Reply:
There have been no instructions at all on how to run unittests. This sections add some possible way an experienced python developer would try/use.

pytest ia able to handle unittests, too, but adds advances features for selecting test-cases, etc.

Reply:
Actually all the tests except (cms tests) are out of date. We are going to rewrite them during porting to py3 and we have started doing that:
https://github.com/sk1project/color-picker/tree/master/src